support/docker/production/.env

```diff
@@ -0,0 +1,15 @@
+DB_USERNAME=postgres_user
+DB_PASSWORD=postgres_password
+PEERTUBE_WEBSERVER_HOSTNAME=my.domain.tld
+PEERTUBE_WEBSERVER_PORT=443
+PEERTUBE_WEBSERVER_HTTPS="true"
+PEERTUBE_REDIS_HOSTNAME=redis
+PEERTUBE_SMTP_USERNAME=null
+PEERTUBE_SMTP_PASSWORD=null
+PEERTUBE_SMTP_HOSTNAME=null
+PEERTUBE_SMTP_PORT=25
+PEERTUBE_SMTP_FROM=noreply@peertube.domain.tld
+PEERTUBE_SMTP_TLS="true"
+PEERTUBE_ADMIN_EMAIL=admin@domain.tld
```

Comment:
I think it would be better to add the comment from https://github.com/Chocobozzz/PeerTube/pull/617/files#diff-5003d902a2333dfdbcd79d688f2806daL38 directly in this file